auraed/src/runtime/cell_service/cells/mod.rs

```diff
@@ -59,7 +59,14 @@ impl CellSpec {
                 cpu_weight: CpuWeight::new(0),
                 cpuset_mems: CpusetMems::new("".into()),
             },
-            shared_namespaces: Default::default(), // nothing shared in default
+            shared_namespaces: SharedNamespaces {
```

Comment:
do you think we should still share nothing by default but share everything just for tests?

Reply:
or implement default for SharedNamespaces that does this (so we're still hiding it behind a "default".

Reply:
The default will still have everything as unshared, which I believe is what is wanted. 

This function is only for the tests, so that we don't need a new `CellSpec` each time. Having all shared namespaces allows the tests to not hang for now, which I think is a better position to be in because I was no longer running the tests prior to opening PRs.

I do think that we want to revisit this and see what the issue was, which ns was the cause, and is there a fix, which is why I updated the todo in the other file.